bigbone-rx/src/main/kotlin/social/bigbone/rx/RxListMethods.kt

```diff
@@ -24,4 +28,90 @@ class RxListMethods(client: MastodonClient) {
             }
         }
     }
+
+    fun getList(listId: String): Single<MastodonList> {
+        return Single.create {
+            try {
+                val list = listMethods.getList(listId)
+                it.onSuccess(list.execute())
+            } catch (throwable: Throwable) {
+                it.onErrorIfNotDisposed(throwable)
+            }
+        }
+    }
+
+    @JvmOverloads
+    fun createList(
+        title: String,
+        repliesPolicy: MastodonList.RepliesPolicy = MastodonList.RepliesPolicy.List
+    ): Single<MastodonList> {
+        return Single.create {
+            try {
+                val list = listMethods.createList(title, repliesPolicy)
+                it.onSuccess(list.execute())
+            } catch (throwable: Throwable) {
+                it.onErrorIfNotDisposed(throwable)
+            }
+        }
+    }
+
+    fun updateList(
+        listId: String,
+        title: String,
+        repliesPolicy: MastodonList.RepliesPolicy
```

Comment:
I think, here we could also set a default value as done on line 46 (and add a JvmOverloads annotation).

Reply:
See other comment.